toponetx/algorithms/spectrum.py

```diff
@@ -305,6 +326,26 @@ def simplicial_complex_adjacency_spectrum(
     return laplacian_spectrum(SC.adjacency_matrix(rank=dim, weight=weight))
 
 
+def path_complex_adjacency_spectrum(
+    PC: PathComplex, dim: int, weight: str | None = None
+):
+    """Return eigenvalues of the adjacency matrix of PC.
+
+    Parameters
+    ----------
+    PC : PathComplex
+    dim : int
+    weight : str, optional
+        If None, then each cell has weight 1.
+
+    Returns
+    -------
+    evals : NumPy array
```

Comment:
Single return values do not have a name according to NumPy's documentation style.

```suggestion
    np.ndarray
```